Test/Unit/Block/JsTest.php

```diff
@@ -0,0 +1,205 @@
+     */
+    public function testGetConnectJsUrl()
+    {
+        // For CDN URL in sandbox mode
+        $this->setSandboxMode();
+        $result = $this->block->getConnectJsUrl();
+        $this->checkEquals($result, true, 'connect.js');
+
+        // For CDN URL in production mode.
+        $this->setSandboxMode(false);
+        $result = $this->block->getConnectJsUrl();
+        $this->checkEquals($result, true, 'connect.js');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function testGetCheckoutKey()
+    {
+        $storeId = 0;
+        $key = 'pKv_pOzRTEST.TESTkEIjTEST.TEST01f0d15501cd7548c1953f6666b2689f2e5a20198c5d7f886c004913TEST';
+        $this->configHelper->expects($this->any())
+            ->method('getAnyPublishableKey')
+            ->will($this->returnValue($key));
+
+        $result = $this->block->getCheckoutKey();
+
+        $this->assertStringStartsWith('pKv_', $result, '"Any Publishable Key" not working properly');
+        $this->assertEquals(strlen($key), strlen($result), '"Any Publishable Key" have invalid length');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function testGetReplaceSelectors()
+    {
+        $value = '.replaceable-example-selector1|append
+.replaceable-example-selector2|prepend,.replaceable-example-selector3';
+
+        $correctResult = [
+            '.replaceable-example-selector1|append .replaceable-example-selector2|prepend',
+            '.replaceable-example-selector3'
+        ];
+
+        $this->configHelper->expects($this->once())
+            ->method('getReplaceSelectors')
+            ->will($this->returnValue($value));
+
+        $result = $this->block->getReplaceSelectors();
+
+        $this->assertEquals($correctResult, $result, 'getReplaceSelectors() method: not working properly');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function testGetGlobalCSS()
+    {
+        $value = '.replaceable-example-selector1 {
+            color: red;
+        }';
+
+        $this->configHelper->expects($this->once())
+            ->method('getGlobalCSS')
+            ->will($this->returnValue($value));
+
+        $result = $this->block->getGlobalCSS();
+
+        $this->assertEquals($value, $result, 'getGlobalCSS() method: not working properly');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function testIsEnabled()
+    {
+        $storeId = 0;
+        $this->configHelper->expects($this->any())
+            ->method('isActive')
+            ->with($storeId)
+            ->will($this->returnValue(true));
+
+        $result = $this->block->isEnabled();
+
+        $this->assertTrue($result, 'IsEnabled() method: not working properly');
+    }
+    /**
+     * Check if CDN Url equal or not.
+     *
+     * @param bool   $sandbox
+     * @param string $file
+     */
+    public function checkEquals($result, $sandbox = true, $file = '')
+    {
+        $mode = ($sandbox) ? HelperConfig::CDN_URL_SANDBOX : HelperConfig::CDN_URL_PRODUCTION;
```

Comment:
IMO test should not include condition. Can you rewrite test above without using this method, eg., 
> $this->assertEquals(HelperConfig::CDN_URL_SANDBOX . DIRECTORY_SEPARATOR . 'track.js', $result);

Reply:
done